eng/common/labels.yaml

```diff
@@ -0,0 +1,117 @@
+# cspell:ignore bfff
+
+issue_kinds:
+  description: "Issue kinds"
+  labels:
+    bug:
+      color: cccccc
+      description: Something isn't working
+    feature:
+      color: cccccc
+      description: New feature or request
+    docs:
+      color: cccccc
```

Comment:
feel like the colors for bug and features could be a little more visibile, at least for bug

Reply:
Not opposed, but fwiw my thinking was that the kind tags should all have the same color since there will only be one and it makes it easier to spot kind when scrolling.

Reply:
yeah, I do feel like its nice when looking at a set of issues to be able to see the bugs pop out a little more

Reply:
feel free to change it!